string frame_a
string frame_b
---
geometry_msgs/Pose pose